rastervision_core/rastervision/core/data/raster_source/multi_raster_source.py

```diff
@@ -19,18 +19,31 @@ class MultiRasterSource(ActivateMixin, RasterSource):
     their output along the channel dimension (assumed to be the last dimension).
     """
 
-    def __init__(self,
-                 raster_sources: Sequence[RasterSource],
-                 raw_channel_order: Sequence[conint(ge=0)],
-                 channel_order: Optional[Sequence[conint(ge=0)]] = None,
-                 crs_source: conint(ge=0) = 0,
-                 raster_transformers: Sequence = []):
+    def __init__(
+            self,
+            raster_sources: Sequence[RasterSource],
+            raw_channel_order: Sequence[conint(ge=0)],
+            force_subchip_size_fill_value: Optional[Union[int, float]] = None,
+            force_same_dtype: bool = False,
+            channel_order: Optional[Sequence[conint(ge=0)]] = None,
+            crs_source: conint(ge=0) = 0,
+            raster_transformers: Sequence = []):
         """Constructor.
 
         Args:
             raster_sources (Sequence[RasterSource]): Sequence of RasterSources.
             raw_channel_order (Sequence[conint(ge=0)]): Channel ordering that
                 will always be applied before channel_order.
+            force_subchip_size_fill_value (Optional[Union[int, float]]):
+                Value used to pad sub-chips so that they are all the same size as the
+                sub-chip from the first sub-source.  This is to accommodate small
+                differences in sub-raster size such those that might be caused by
+                attempting to reproject disparate sub-rasters into the same projection.
+                No special reprojection logic is triggered by this parameter.  Use with
+                caution.
```

Comment:
Isn't the output of `RasterSource._get_chip()` supposed to be the same size as specified by the window it takes in?

Reply:
Well, the size of the overall chip ends up being the same size as the first subchip of the multiraster source.

In most instances, this will have no effect at all.  The idea here is to handle cases where there are multiple rasters which cover the same area but (in my case) differ by 1 in their width.  When that case is encountered, the extra column will be filled with the fill value in subchip from the smaller raster.

Reply:
Looking at the code for `RasterioSource`, it seems like `window` _is_ supposed to be in pixel coordinates and `RasterioSource` returns a chip of the height and width specified by the `window`.

Disabling the validation check for equal extents makes sense though.